### PR TITLE
Move default args to CMD

### PR DESCRIPTION
**Pull request by teohhanhui**
https://github.com/docker-library/official-images/pull/4404#issuecomment-408252085

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,42 +54,6 @@ $ docker run --name my-running-varnish --tmpfs /usr/local/var/varnish:exec -d -p
 
 Then you can hit `http://localhost:8080` or `http://host-ip:8080` in your browser.
 
-## Advanced configuration using environment variables
```
Comment (dunglas) [inline README.md]: Shouldn't this doc be updated instead of deleted?

Comment (teohhanhui) [reply README.md]: I think we need to restructure the entire "How to use this image" section anyway. So removing for now...

Comment (teohhanhui) [reply README.md]: We could follow the structure of https://github.com/docker-library/docs/blob/master/haproxy/README.md#how-to-use-this-image

**Comment by teohhanhui** [comment]
@jewome62 Yes, I think we should only have 1 example showing how to pass your own options to `varnishd`. No need for separate examples for port number, VCL path, cache size, etc.

**Pull request merged**